lib/rules/font-family-no-missing-generic-family-keyword/README.md

```diff
@@ -53,3 +53,20 @@ a { font: inherit; }
 ```css
 a { font: caption; }
 ```
+
+## Optional secondary options
+
+### `ignoreFontFamilies: : ["/regex/", /regex/, "string"]`
+
+Given:
+
+```
+["custom-icon-font"]
+```
+
+The following pattern is _not_ considered a violation:
+
+<!-- prettier-ignore -->
+```css
+a { font-family: custom-font; }
+```
```

Comment:
[suggestion] I feel more understandable if a violation example is added, for example:

---

Given:

```
["custom-font"]
```

The following pattern is _not_ considered a violation:

```css
a { font-family: custom-font; }
```

The following pattern is considered a violation:

```css
a { font-family: invalid-custom-font; }
```